chainstate/src/detail/tokens/mod.rs

```diff
@@ -48,6 +59,65 @@ pub fn check_tokens_burn_data(
     Ok(())
 }
 
+pub fn check_nft_issuance_data(
+    chain_config: &ChainConfig,
+    issuance: &NftIssuanceV1,
+    tx_id: Id<Transaction>,
+    source_block_id: Id<Block>,
+) -> Result<(), TokensError> {
+    // Ticker
+    check_token_text_length(
+        &issuance.metadata.ticker,
+        chain_config.token_max_ticker_len(),
+    )
+    .map_err(|_| TokensError::IssueErrorInvalidTickerLength(tx_id, source_block_id))?;
+
+    check_is_text_alphanumeric(&issuance.metadata.ticker).map_err(|_| {
```

Comment:
Same here... you return in that function a specific error, which you override and ignore. Isn't it better to return a boolean and then convert that to an error if need be?

Reply:
Move the discussion here: https://github.com/mintlayer/mintlayer-core/pull/432#discussion_r987678827